### ISSUE TITLE
code example misses :$id

### ISSUE THREAD

**Issue by melezhik** [Leont/protocol-postgres#1]
Probably code example misses :$id ?

right  now:
```raku
whenever $resultset.hash-rows -> (:$name, :$description) {
```
see failed build - https://ci.sparrowhub.io/report/2478 (`Unexpected named argument 'id' passed in sub-signature`)

should be:

```
whenever $resultset.hash-rows -> (:$name, :$description, :$id) {
```
see successful build - https://ci.sparrowhub.io/report/2479

**Comment by Leont** [comment]
That's a good point!

**Comment by Leont** [comment]
This was fixed in 0.0.5
